.github/workflows/ci-workflow.yml

```diff
@@ -0,0 +1,131 @@
+
+      - name: Copyright Check
+        run: ./copyrightCheck.sh             
+
+      - name: Install Java8
+        uses: actions/setup-java@v2
+        with:
+          java-version: '8'
+          distribution: 'adopt-hotspot'
+          cache: 'gradle'      
+
+      - name: Set GitHub Tag environment variable
+        if: startsWith(github.ref, 'refs/tags/v') == true
+        run: |
+          TAG_ID=${GITHUB_REF##*/}
+          echo "ACTIONS_TAG=$TAG_ID" >> $GITHUB_ENV
+
+      - name: Gradle build
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          ./gradlew clean build
+
+      - name: Run Smoke Tests
+        run: |
+          ./run-smoketests.sh
+
+      - name: Install Ruby 2.6.5
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 2.6.5
+          bundler-cache: false
+
+      - name: Install Ruby Gems, Run IVT
+        run: |
+          gem install bundler
+          BUNDLE_GEMFILE="./test/Gemfile" bundle install
+          gem specific_install -l https://github.com/Alvearie/hri-test-helpers.git main
+          curl -sL https://ibm.biz/idt-installer | bash
+          ibmcloud login --apikey $CLOUD_API_KEY -r us-east || { echo 'IBM Cloud CLI login failed!'; exit 1; }
+          ibmcloud plugin install event-streams
+          ibmcloud es init -i ${ES_INSTANCE}
+          ./run-ivttests.sh
+
+      - name: Run Nightly Tests
+        if: github.ref == 'refs/heads/develop' && github.event_name == 'workflow_dispatch'
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          ibmcloud ks cluster config --cluster hri-dev1-cluster-1
+          ./gradlew copyNightlyTestDependencies -PcloudApiKey=$CLOUD_API_KEY
+          ./run-nightlytests.sh
+
+      - name: Upload Test Results
+        if: github.ref == 'refs/heads/main' || github.ref == 'refs/heads/develop'
+        run: |
+          ./test/spec/upload_test_reports.rb IVT
+          ./test/spec/upload_test_reports.rb Nightly
+
+#      TODO: Add SonarCloud (Static Code) scan after permissions issue is fixed
+#      - name: SonarCloud Scan
+#          uses: sonarsource/sonarcloud-github-action@v1.6
+#          with:
+#            projectBaseDir: ./src
+#          env:
+#            SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+
+      - name: Gradle publish
+        if: ${{ github.ref == 'refs/heads/develop' }}
+        run: |
+          ./gradlew publish
+
+      - name: Post Slack Update
+        if: ${{ job.status == 'failure' && ( github.ref == 'refs/heads/main' || github.ref == 'refs/heads/develop' || startsWith(github.ref, 'refs/heads/support-') ) }}
+        id: slack
+        uses: slackapi/slack-github-action@v1.14.0
+        with:
+          payload: "{\"Repo\":\"${{ github.repository }}\",\"Workflow\":\"${{ github.workflow }}\",\"Branch\":\"${{ env.BRANCH_NAME }}\",\"Link\":\"https://github.com/Alvearie/hri-mgmt-api/actions/runs/${{ github.run_id }}\"}"
```

Comment:
payload has an "hri-mgmt-api" in it. Come to think of it, maybe I should've plugged in "github.repository" in that value.

Reply:
Good Catch - Fixed now.